### ISSUE TITLE
1.0.x release 

### ISSUE THREAD

**Issue by mitermayer** [rsocket/rsocket-java#284]
I was wondering what is outstanding for doing a 1.0.x release to maven central ?

**Comment by robertroeser** [comment]
Need to add leasing support back in, support resumeablity, and probably have support for extension frames. The current version is 0.9-SNAPSHOT, we could change that to 0.9.0 and release that to maven central.

**Comment by mitermayer** [comment]
@robertroeser I like the idea of releasing 0.9.0 to maven central now, this would make it easier to keep track on changes from 0.9.0 to 1.0.0.

**Comment by yschimke** [comment]
I think just blocked on https://github.com/rsocket/rsocket-java/issues/363

Otherwise based on current usage, I would happily release a 1.0 version.

**Comment by yschimke** [comment]
0.9.8 released

https://jcenter.bintray.com/io/rsocket/rsocket-core/0.9.8/

1.0.0 is just a github release away

**Comment by yschimke** [comment]
Thing's I'd like supported for 1.0

- leases (nice to have)
- resumablity (nice to have)
- Aeron - moved to an external library or fixed
- TCK moved externally

**Comment by yschimke** [comment]
Leases are being worked on https://github.com/rsocket/rsocket-java/pull/421

**Comment by yschimke** [comment]
This isn't the right way to track missing features.  These should just be milestones on issues.

publishing already covered here https://github.com/rsocket/rsocket-java/issues/463

**Comment by yschimke** [comment]
Resume support https://github.com/rsocket/rsocket-java/issues/339
Aeron split out or fix https://github.com/rsocket/rsocket-java/issues/309
Lease support https://github.com/rsocket/rsocket-java/issues/284
